test/e2e/adminapi_cluster_getlogs.go

```diff
@@ -45,21 +45,23 @@ func testGetPodLogsOK(ctx context.Context, containerName, podName, namespace str
 	expectedLog := "mock-pod-logs"
 
 	By("creating a test pod in openshift-azure-operator namespace with some known logs")
-	pod := mockPod(containerName, podName, namespace, expectedLog)
-	pod, err := clients.Kubernetes.CoreV1().Pods(namespace).Create(ctx, pod, metav1.CreateOptions{})
-	Expect(err).NotTo(HaveOccurred())
+	podDefinition := mockPod(containerName, podName, namespace, expectedLog)
```

Comment:
do we need to maintain a separate reference to the old definition after creation here?

Reply:
we don't ever use `podDefinition` after creating the `pod`, and they share the same type (pointer to `corev1.Pod`) so I think the old implementation where we reuse the same variable name/pointer makes more sense.

Reply:
I thought that's what you meant. I find this re-using of the same variable name weird but I think I just need to adjust to it :D. I'll make that change now

Reply:
like I said, it's a minor nit and I'm fine with it merging as-is, just want to make sure I wasn't missing any specific intent here for handling the two values separately.

Reply:
No no. I'm keen to get find the standard we have so do go for it with conventions and things like this.

Ty